contracts/Registry/TokenCuratedRegistry.sol

```diff
@@ -85,4 +84,13 @@ contract TokenCuratedRegistry is StakedRegistry, TimelockableItemRegistry {
     require(exists(id));
     return address(challenges[id]) != 0x0;
   }
+
+  // Removes an item and all state related to the item
+  function _reject(bytes32 id) internal {
+    ownerStakes[id] = 0;
+    delete owners[id];
+    delete unlockTimes[id];
+    delete challenges[id];
+    _remove(id);
```

Comment:
still feel kindaaa weird about the `_remove`...buuut, not weird enough that it's a blocker!